examples/CMakeLists.txt

```diff
@@ -1,22 +1,19 @@
-install(DIRECTORY DESTINATION ${CMAKE_INSTALL_PREFIX}/examples)
+# As part of installation we'll copy the examples
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+include(X3D_ADD_EXAMPLE)
+# Tests are in the folder tests
+# This is only to have in the installation director the examples
+set(examples_dir "${CMAKE_INSTALL_PREFIX}/examples")
+install(DIRECTORY DESTINATION ${example_dir})
 
-# if testing active create a working dir for testing
-set(test_dir "${PROJECT_BINARY_DIR}/RunTests")
-file(MAKE_DIRECTORY ${test_dir})
-
-# We need to find a way to check if 2DECOMP&FFT is build with ADIOS2
-option(USE_ADIOS2 "Build XCompact with ADIOS2 library" OFF)
-
-if (${BUILD_TESTING_FULL})
-  add_subdirectory(ABL-Atmospheric-Boundary-Layer)
-  add_subdirectory(Cavity)
-  add_subdirectory(Channel)
-  if(NOT ${USE_ADIOS2})
-    add_subdirectory(Cylinder-wake)
-  endif()
-  add_subdirectory(Gravity-current)
-  add_subdirectory(Mixing-layer)
-  add_subdirectory(MHD)
-  add_subdirectory(TBL-Turbulent-Boundary-Layer)
-  add_subdirectory(Wind-Turbine/NREL-5MW_ALM)
-endif (${BUILD_TESTING_FULL})
+add_subdirectory(ABL-Atmospheric-Boundary-Layer)
+add_subdirectory(Cavity)
+add_subdirectory(Channel)
+add_subdirectory(Cylinder-wake)
+add_subdirectory(Gravity-current)
+add_subdirectory(MHD)
+add_subdirectory(Mixing-layer)
+add_subdirectory(Periodic-hill)
+add_subdirectory(TBL-Turbulent-Boundary-Layer)
+add_subdirectory(TGV-Taylor-Green-vortex)
+#add_subdirectory(Wind-Turbine)
```

Comment:
Why leave out the wind turbine examples?

Reply:
I'll add it as well. Just forgot

Reply:
I'll add back the Wind-Turbine in a dedicated PR. Because of  many additional files and directories the install bit requires some modifications.